### PR TITLE
Launch Hack OC

### PR DESCRIPTION
**Pull request by YodaLightsabr**
Launch the Hack OC website

To-do:

- [x] Open registrations
- [x] Improve site content
   - [x] Adjust copy
   - [x] Add information cards
   - [x] Add question cards
- [ ] Improve analytics
- [x] Add Discord integration to the registration form

### CODE REVIEW

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:


| [**/**](https://hackoc-site-4qd52pu1x.hackclub.dev/) | [**/register**](https://hackoc-site-4qd52pu1x.hackclub.dev/register) |
|:-:|:-:|
|<a href="https://hackoc-site-4qd52pu1x.hackclub.dev/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/-5nGlTV4tQhLyxwI0Xlxl9Dr64YmCygLKZv-bdATNGvs8idvPiRO38FJ4Vn5mrqayUCmNEDb5M8RT0D7jXqK6g.png">(view full size)</a> |<a href="https://hackoc-site-4qd52pu1x.hackclub.dev/register">Screenshot of /register</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/0wAzYvmD5RalGMUj4zkcGX51tHHIUkiItRB0cuZWAYVW94AODDUiZj2maP9sN0TzJecxtVT3nny9bn_x21Q7Qw.png">(view full size)</a> |



| [**…ration/discord-success**](https://hackoc-site-4qd52pu1x.hackclub.dev/registration/discord-success) |
|:-:|
|<a href="https://hackoc-site-4qd52pu1x.hackclub.dev/registration/discord-success">Screenshot of /registration/discord-success</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/wFyALY2GkTG6Dklm92lv4Wu2nlMPB75tFKQ5uUwWFNYEiN2RIZGBzO-nLnwN1hcGPesLMAdiUe-A_cfUFtohFA.png">(view full size)</a> |



And 1 other route:
- [**/api/discord/link**](https://hackoc-site-4qd52pu1x.hackclub.dev/api/discord/link)



Commit 64e296b52c4aeb0880dcdcddc3b265894ba0e6e7 (https://hackoc-site-4qd52pu1x.hackclub.dev).

File: pages/index.js [hunk 1]
```diff
@@ -243,35 +321,144 @@ export default function Home() {
...
+          gridGap: cardGap,
+          padding: '30px',
+          margin: '0 auto',
+          maxWidth: `calc(calc(${cardSize} * 2) + ${cardGap} + 60px + 200px)`
+        }}>
+          
+            <Card title="Build something" icon="idea">
+              By the end, you’ll have built something new and unique with your team.
+            </Card>
+            <ImageCard src="/social.png" caption="Placeholder image" />
```
Comment (robert-goll) [inline pages/index.js]: Do we want to keep the placeholder image caption

Comment (YodaLightsabr) [reply pages/index.js]: Just fixed!

**Comment by YodaLightsabr** [comment]
Requesting a review but merging now to test Discord OAuth

**Pull request merged**